plugins/woocommerce/legacy/js/admin/system-status.js

```diff
@@ -48,17 +49,20 @@ jQuery( function ( $ ) {
 
 						// Format value
 						var the_value   = $value_html.text().trim();
-						var value_array = the_value.split( ', ' );
 
-						if ( value_array.length > 1 ) {
-							// If value have a list of plugins ','.
-							// Split to add new line.
-							var temp_line ='';
-							$.each( value_array, function( key, line ) {
-								temp_line = temp_line + line + '\n';
-							});
-
-							the_value = temp_line;
+						if ( export_type != 'active-plugins' && export_type != 'inactive-plugins' && export_type != 'dropin-plugins' && export_type != 'must-use-plugins') {
+							var value_array = the_value.split( ', ' );
+	
+							if ( value_array.length > 1 ) {
+								// If value have a list of plugins ','.
+								// Split to add new line.
+								var temp_line ='';
+								$.each( value_array, function( key, line ) {
+									temp_line = temp_line + line + '\n';
+								});
+	
+								the_value = temp_line;
+							}
```

Comment:
```suggestion
						
```

I think we can just get rid of this line break logic. Status report looks fine without it as well, then you won't need to add all the data attributes above as well.